srv/salt/_modules/cephprocesses.py

```diff
@@ -38,8 +38,14 @@
              'client-radosgw': [],
              'master': []}
 
+# Processes like lrbd have an inverted logic
+# if they are running it means that the service is _NOT_ ready
+# as opposed to the the services in the 'processes' map.
+processes_inverted = {'igw': ['lrbd']}
```

Comment:
I'm just bikeshedding here, but I think absent_processes might match the intention more.

Reply:
Ah, yes. That might make it more clear. Changing